assets/src/js/experiments/index.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Admin experiments.
+ *
+ * @package    Secure Custom Fields
+ * @since      6.4.3
+ */
+
+interface ExperimentsData {
+	[ key: string ]: boolean;
+}
+
+interface ExperimentsObject {
+	[ key: string ]: boolean | ( ( name: string ) => boolean );
+	isEnabled: ( name: string ) => boolean;
+}
+
+interface ACF {
+	experiments?: ExperimentsObject;
+	[ key: string ]: any;
+}
+
+declare global {
+	interface Window {
+		acf: ACF;
+		acfExperiments?: ExperimentsData; // Variable created by wp_localize_script in admin-experiments.php L150.
+	}
+	var acf: ACF;
+	var acfExperiments: ExperimentsData | undefined;
+}
+
+// Create a module to avoid global scope augmentation issues.
+export {};
```

Comment:
Can you elaborate on why export is necessary here?

Reply:
Can you move types to another file to solve the issue? The rest of the code is contained inside IIFE which prevents variables from being exposed in the global namespace.

Reply:
Removed all JS of the experiment :)